src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -67,6 +67,10 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     } catch (error) {
       logger.error(error.message);
     }
+    const supportedConstraints = navigator.mediaDevices.getSupportedConstraints();
```

Comment:
Sorry, I missed it in the previous revision. Could you put your line into the above `try/catch` block between line 66 and 67?
In a non-secured website, `navigator.mediaDevices` will be `undefined`.